sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -154,27 +168,28 @@ namespace Azure { namespace Core {
     }
 
     /**
-     * @brief Get a deadline associated with this context or the branch of contexts this context
+     * @brief Gets the deadline for this context or the branch of contexts this context
      * belongs to.
      *
-     * @return A deadline associated with the context found; `Azure::DateTime::max()` value if a
-     * specific value can't be found.
+     * @return The deadline associated with the context;
+     * @return `Azure::DateTime::max()` if no deadline is specified.
```

Comment:
it is `@retval` that would handle multiple return values. `@return` won't.
I suggest we use `@return` here as before.
With `@retval` I am not sure what to put for the value before describing it.